src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -1553,8 +1553,11 @@
   "com.minecolonies.coremod.jei.chancenegskill.tip": "§e%d%% base chance §c- skill modifier",
   "com.minecolonies.coremod.jei.conditions.tip": "§bSpecial conditions apply",
   "com.minecolonies.coremod.jei.baker": "Bakes bread, and other things at higher levels. Can be taught extra recipes starting at level 3.",
+  "com.minecolonies.coremod.jei.beekeeper": "Takes care of beehives, harvesting either honey or honeycomb.",
```

Comment:
@ravenbuilder934

Reply:
I came _very_ close to making a "not the bees!" reference there, but I managed to refrain.